docs/docsource/Operation-Architecture/Zenko_CLI/Object_Lifecycle_Management.rst

```diff
@@ -98,4 +98,47 @@ for more examples and explanations on lifecycle rules.
        $ aws s3api get-bucket-lifecycle-configuration --bucket zenko-bucket
 
 Once the lifecycle rules on the bucket are set, the rules apply to all
-objects in the specified bucket.
\ No newline at end of file
+objects in the specified bucket.
+
+.. _Lifecycle Queries:
+
+Querying Lifecycle Events
+-------------------------
+
+A lifecycle metrics feature is not available for release 1.1. The only
+available lifecycle event metric is to send a head-object request to
+Zenko's CloudServer S3 API.
+
+Querying the CloudServer requires an active kubectl session with the
+Zenko controller and S3 API functionality configured as described in
+:ref:`S3 API config`. Once this is configured, use the head-object command
+as described in
+https://docs.aws.amazon.com/cli/latest/reference/s3api/head-object.html.
+
+For example:
+
+.. code::
+   
+   $ aws s3api head-object --bucket mybucket-1 --key myobjects1 --endpoint http://10.0.0.1
+
+returns:
+
+.. code::
+
+   {
+      "AcceptRanges": "bytes",
+      "ContentType": "application/octet-stream",
+      "LastModified": "Tue, 16 Apr 2019 22:12:33 GMT",
+      "ContentLength": 1,
+      "ETag": "\"e358efa489f58062f10dd7316b65649e\"",
+      "StorageClass": "aws-backend",
```

Comment:
~aws-backend~ `aws-storage-location`

That way we're more consistent with the naming scheme in Orbit which terms this a "storage location".

Reply:
done